### ISSUE TITLE
"Delete" Option for Filament Inventory Management

### ISSUE THREAD

**Issue by Aveplatter** [Filament-uno/site#9]
I'd like to be able to delete filament rolls from my inventory. Currently, the only option is to mark the roll as empty, which is not the same thing. Sometimes I make mistakes when I enter in new rolls.

**Comment by paul999** [comment]
Yes, those are definitely different. Need to think how to handle statistical data, but will add it, and will also add it to the apps.

**Comment by paul999** [comment]
I have tried if I was able to quickly implement this, however due to the way statistics and everything is saved right now, it makes it a bit hard to do. I will need to do some rethinking on how I store stuff, however that might take a bit before it is implemented.
I have added pretty high on the list to fix, but can't give any promises yet when it arrives.

**Comment by paul999** [comment]
I have implemented two delete type options. On the site, you can now delete a roll, and choose for keeping the statistics (this will internally mark it as deleted, but all statistics for things like the graphs on the profile will be kept). You can also choose to also delete the statistics. This will basically delete everything related to the inventory item.

Currently, this is only available on the site. Right now, I ran out of time to also implement it in the apps. This should hopefully follow soon in the next revision of the apps. Hopefully I didn't break anything :).

**Comment by Aveplatter** [comment]
sweet!

**Comment by paul999** [comment]
I have just submitted an update for the iOS app to Apple. With a bit of luck Apple approves it later this weekend.
Android should follow soon as well :).
